packages/custom-elements/gulpfile.js

```diff
@@ -26,7 +27,7 @@ gulp.task('default', () => {
       language_out: 'ECMASCRIPT5_STRICT',
       externs: ['externs/custom-elements.js'],
       dependency_mode: 'STRICT',
-      entry_point: ['/src/custom-elements'],
+      entry_point: ['/lib/custom-elements'],
```

Comment:
`src/native-shim.js` probably needs to stay where it is since that's the only way to reference it if you're using @webcomponents/custom-elements directly. The only other one that might reasonably be used this way is this entry point. I need to think a bit more about this.

Reply:
It seems needlessly risky to break backcompat just for the sake of pretty filesystem paths. I moved the ts sources to ts_src/ (better name ideas welcome and easy to change) and the typescript compiler's output goes to src/